scripts/logistic_regression_clf.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import os
+
+import joblib
+from datasets import load_dataset
+from loguru import logger
+from sklearn.linear_model import LogisticRegression
+
+
+def main(args):
+    logger.info(f"Load dataset from {args.dataset_name}")
+    dataset = load_dataset(args.dataset_name)
+    dataset = dataset.class_encode_column("class")
+
+    train_ds = dataset["train"]
```

Comment:
Merge train and validation

```suggestion
    train_ds = concatenate_datasets(dataset["train"],dataset["validation"]) 
```